src/passes/rejectUnsupportedFeatures.ts

```diff
@@ -93,9 +92,6 @@ export class RejectUnsupportedFeatures extends ASTMapper {
     }
     this.visitExpression(node, ast);
   }
-  visitInlineAssembly(node: InlineAssembly, _ast: AST): void {
-    this.addUnsupported('Yul blocks are not supported', node);
-  }
```

Comment:
It's done in the `YulTransformer` class

Reply:
Can you make it the same way as `RejectUnsupportedPass` which collects all the errors first and at the end it throws them all. Since this pass is going to be run after `Reject`, you could define error collection there, reusing function in this class